backend/src/nodes/properties/outputs/pytorch_outputs.py

```diff
@@ -1,10 +1,51 @@
-from .. import expression
+try:
+    import torch
+except:
+    torch = None
+
+from typing import Union
+
+from ...properties import expression
```

Comment:
Why did you change the expression import? Is that style preferred?

Reply:
I had removed it by accident and added it again when I realized I needed it, and didn't realize it was different